tests/test_det_profile.py

```diff
@@ -64,5 +69,16 @@ def test_make_semicoax(test_data_configs):
 
 
 def make_v07646a():
```

Comment:
I'm not sure test functions that do not start with `test_` are discovered by pytest...

Reply:
Forgot to change it after merging